astro-docs/src/content/docs/technologies/dotnet/Guides/migrate-from-nx-dotnet-core.mdoc

```diff
@@ -0,0 +1,448 @@
+
+**Migration path:**
+
+1. **If you disabled inference to exclude certain projects:**
+
+   - Use the `exclude` property in the plugin configuration:
+     ```json
+     {
+       "plugins": [
+         {
+           "plugin": "@nx/dotnet",
+           "exclude": ["legacy-projects/**", "temp-projects/**"]
+         }
+       ]
+     }
+     ```
+
+2. **If you disabled inference to use explicit project.json files:**
+
+   - The new plugin will still detect your projects via `.csproj` files
+   - You can keep `project.json` files to customize targets, but they're optional
+   - Consider removing them to use the automatically inferred targets
+
+3. **If you need to completely hide projects from Nx:**
+   - Use the `exclude` property in the plugin configuration
+   - Or move projects outside the workspace
+
+## Configuration Changes
+
+### Module Boundaries → Nx Conformance
+
+If you used `moduleBoundaries` in your @nx-dotnet/core config, you may want to migrate to [Nx Conformance rules](https://nx.dev/docs/enterprise/powerpack/conformance)
+
+The script used to evaluate module boundaries is missing several features from the eslint and conformance implementations, and is not present in the new plugin. If you wish to continue using it, you'll need to inline the implementation in your repository and update the path in `Directory.build.targets`. Otherwise, you should remove the target from `Directory.build.targets`.
+
+### NuGet Package Management
+
+Replace the `sync` generator with MSBuild's [Central Package Management](https://learn.microsoft.com/nuget/consume-packages/central-package-management).
+
+Create `Directory.Packages.props` at workspace root:
+
+```xml
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageVersion Include="Microsoft.Extensions.Logging" Version="8.0.0" />
+  </ItemGroup>
+</Project>
+```
+
+Update project files to remove version attributes:
+
+```xml
+<!-- Before -->
+<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+
+<!-- After -->
+<PackageReference Include="Newtonsoft.Json" />
+```
+
+## Testing the Migration
+
+Run these commands to verify everything works:
+
+```shell {% frame="none" %}
+# Clear cache
+nx reset
+
+# Verify project detection
+nx show projects
+
+# Test builds
+nx run-many -t build
+
+# Test affected builds
+nx affected -t build
+
+# Run tests
+nx run-many -t test
+```
+
+## Troubleshooting
+
+### Projects Not Detected
+
+**Symptom**: Some .NET projects don't appear in `nx show projects`
+
+**Solution**:
+
+- Ensure project files have `.csproj`, `.fsproj`, or `.vbproj` extensions
+- Check that the plugin is registered in `nx.json`
+- Run `nx reset` to clear the cache
+
+### Build Errors: Missing Dependencies
+
+**Symptom**: Build fails with "Could not find project reference"
```

Comment:
suggestion: people will probably search for these errors specifically so might want to include them in the title if possible to improve their search rankings.